microsetta_interface/templates/submit_interest.jinja2

```diff
@@ -113,7 +133,15 @@ $(document).ready(function(){
         active_section = new_active_section;
     }
 
-    $.validator.addMethod("countryValidation", function(value, element) {
+    jQuery.validator.addMethod("over18", function(value, element) {
```

Comment:
Let's punt for now, but I believe this should be campaign specific

Reply:
Just opened an issue to reflect that - #253

Reply:
Just opened an issue to reflect that - #253

Reply:
Thanks!